n64.mk

```diff
@@ -1,5 +1,6 @@
 BUILD_DIR ?= .
 SOURCE_DIR ?= .
+N64_DFS_OFFSET ?= 1M # Override this to offset where the DFS file will be located inside the ROM
```

Comment:
I wonder if `?=` does make a difference here?

Reply:
Yeah, you can't override on your Makefile if it's not there (at least it didn't work on my tests).

Reply:
Interesting b/c we can override the title anyways, merging this one

Reply:
Hmm I think the difference is in how it is used. One option is setting the var like a dependency and the other is as an individual statement